epictrack-api/src/api/reports/resource_forecast_report.py

```diff
@@ -243,6 +243,48 @@ def _fetch_data(self, report_date: datetime):  # pylint: disable=too-many-locals
             .subquery()
         )
 
+        works_started = (
```

Comment:
curious , why not use 
 db.session.query(wp_alias.work_id)
same like use filter instead of where?
any reason to not to go for sqlalchemy?
i might be totally wrong as well..

Reply:
`query` object has no `scalars` method. Without it we will be getting a list of tuples like, `[(id1,), (id2, )...]` and will need to do a list comprehension on it to get a list of ids, (`[x[0] for x in query_result]`). This way, we get a proper list of ids, like, `[id1, id2, ...]` as a result of the db operation itself. **(At least I am not aware of any methods to convert query object into a list of ids without additional actions after db query)**

Reply:
yeah..i thought so...
dont think we have a scalar for this purpose..the only way is do a list comprehension after the tuple..
we will have to do a careful analysis when we opt for plain queries..when it grows bigger , we will loose the flexibility of the sqlalchmey..  probably we can do a list comprehension to do a tuple to list... 
But i this case ,query is very concise , so its all good...